src/EA.Weee.Core/DirectRegistrant/SellingTechniqueType.cs

```diff
@@ -10,6 +10,9 @@ public enum SellingTechniqueType
         IndirectSellingToEndUser = 1,
 
         [Display(Name = "Direct selling to end user (mail, order, internet etc) and Indirect selling (other)")]
-        Both = 2
+        Both = 2,
+
+        [Display(Name = "Online Marketplaces and Fulfilment Houses")]
```

Comment:
Hmmm --- this seems messy -- there are two existing entries with a Both as well...but now you're adding a third. So what would Both mean in this context? Should it be possible to have all 3? In which case, should this actually be set up as a flags enum?

Reply:
However, I can see that might be a pain because it's used directly in the schema, and I'm not sure if minor version increase allows existing things to change, so I've decided against this for now.